DUI3-DX/Connectors/Autocad/Speckle.Connectors.AutocadShared/HostApp/Extensions/EntityExtensions.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Autodesk.AutoCAD.DatabaseServices;
+
+namespace Speckle.Connectors.Autocad.HostApp.Extensions;
+
+public static class EntityExtensions
+{
+  /// <summary>
+  /// Adds an entity to the autocad database model space record
+  /// </summary>
+  /// <param name="entity">Entity to add into database.</param>
+  /// <param name="layer"> Layer to append object.</param>
+  public static ObjectId Append(this Entity entity, string? layer = null)
```

Comment:
This method's logic is slightly odd. We return `ObjectId.Null` but is that an error we're swallowing? Should we prefer throwing an exception here instead?

Reply:
I'm not keen on this as an extension as it has a dependency on document which is tightly binding us to the static instance